tests/components/MCP3428/test.esp32-c3-idf.yaml

```
@@ -0,0 +1,12 @@
+i2c:
+  - id: i2c_mcp3428
+    scl: 5
+    sda: 4
+
+mcp3428:
+  address: 0b1101000
```

Please use hex here to keep consistent with all other i2c examples.
```suggestion
  address: 0x68
```